lighthouse-core/audits/content-width.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('./audit');
+
+class ContentWidth extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Mobile Friendly',
+      name: 'content-width',
+      description: 'Content is sized correctly for the viewport',
+      requiredArtifacts: ['ContentWidth']
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    if (typeof artifacts.ContentWidth === 'undefined' ||
+        typeof artifacts.ContentWidth.scrollWidth === 'undefined' ||
+        typeof artifacts.ContentWidth.viewportWidth === 'undefined') {
+      return ContentWidth.generateAuditResult({
+        value: false,
+        debugString: 'Unable to find scroll and viewport widths.'
+      });
+    }
+
+    const widthsMatch =
+        artifacts.ContentWidth.scrollWidth === artifacts.ContentWidth.viewportWidth;
```

Comment:
@paullewis you'll get a kick out of how PSI does this: https://goto.google.com/yywmk

your check is quite refreshing, by contrast.